### DISCUSSION TITLE
When does the fix for https://github.com/moby/moby/pull/45992 PR will be released?

### DISCUSSION THREAD

**Discussion by dievilz** [moby/moby#46160] [General]
Pretty much the title.
My testing Ubuntu server has the issue resolved by that PR and I want to know when will be released or, if you could tell me, how can I pull it to my server so I can fix the issue.

docker --version: Docker version 18.03.1-ce, build 9ee9f40

Thanks in advance.

**Comment by cpuguy83** [comment]
The version of Docker you are using is very old (5 years).
If the change is backported it will not be backported to to 18.03.

I did ask on the PR if that needs to be bacported, but again it won't happen to 18.03.
Probably the oldest version would be 23.0.

**Reply by dievilz** [reply]
I see, so what would be my alternatives to get that fix as soon as possible?

**Reply by dievilz** [reply]
Or would you at least tell me please when the fix is planned to be released?
To upgrade to that version as soon as I can

**Reply by akerouanton** [reply]
We will consider backporting this PR only after 25.0 is released. The reason is that, unfortunately we lack confidence it won't break anything. Citing @thaJeztah (see [here](https://github.com/moby/moby/pull/45992#issuecomment-1648054889)):

> This is indeed one of the "historic" areas where a lot of context has been lost, and behavior is very under-defined. Let's get this one in for the 25.0 release at least, and if the world doesn't explode, we can consider backporting to 24.0 and 23.0 branches if needed.

So if you really need it, you're on your own at this point. That is, you have to backport it ontop of the version branch you want and you've to compile dockerd by yourself with the right `make` target, and make the appropriate tests to make sure it doesn't break anything.

**Reply by dievilz** [reply]
Thank you very much for the replies!
Waiting for that 25.0 release
